Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -7640,6 +7640,18 @@ bool CStaticFunctionDefinitions::GetSoundLength(CClientPlayer& Player, double& d
     return false;
 }
 
+bool CStaticFunctionDefinitions::SetSoundLooped(CClientSound& Sound, bool bLoop)
+{
+    Sound.SetLooped(bLoop);
+    return true;

```

Comment:
Don't forget to return the result of Sound.SetLooped (because it can fail, as mentioned in a previous comment).